exonum/src/blockchain/transaction.rs

```diff
@@ -49,10 +100,10 @@ pub type TransactionResult = Result<(), TransactionError>;
 /// See also [the documentation page on transactions][doc:transactions].
 ///
 /// [doc:transactions]: https://exonum.com/doc/architecture/transactions/
-pub trait Transaction: Message + ExonumJson + 'static {
-    /// Verifies the internal consistency of the transaction. `verify` should usually include
-    /// checking the message signature (via [`verify_signature`]) and, possibly,
-    /// other internal constraints. `verify` has no access to the blockchain state;
+pub trait Transaction: ::std::fmt::Debug + Send + 'static + ::erased_serde::Serialize {
```

Comment:
Unless `Debug` is directly required for `Transaction` trait, I'd prefer removing it to keep trait as small as possible. `Debug` implementations should be enforced by `#[deny(missing_debug_implementation)]`, not by user interface.

Reply:
We discussed this before, and choose to require `Debug` from user.
This will be used with `:erased_serde::Serialize` to give end-user info about transaction.